cookbooks/travis_packer_templates/spec/attributes_spec.rb

```diff
@@ -0,0 +1,4 @@
+describe file('/.node-attributes.yml'), dev: true do
```

Comment:
images in which packer-templates itself is being tested, DO NOT have this location (booting from current prod CI images)